src/librustc_codegen_llvm/llvm_util.rs

```diff
@@ -307,16 +307,16 @@ pub fn target_features(sess: &Session) -> Vec<Symbol> {
         .collect()
 }
 
-pub fn target_feature_whitelist(sess: &Session) -> &'static [(&'static str, Option<Symbol>)] {
+pub fn target_feature_allowlist(sess: &Session) -> &'static [(&'static str, Option<Symbol>)] {
```

Comment:
Nit: `allowed_target_features` reads a bit better to me.

Reply:
Going by my previous comment, `supported_target_features` might work well.